### ISSUE TITLE
German locale breaks the UI

### ISSUE THREAD

**Issue by mnemanja** [mui-org/material-ui-pickers#1292]
[A GIF or MEME to give some spice of the internet](https://media.giphy.com/media/5Zesu5VPNGJlm/source.gif)

## Environment

| Tech                 | Version |
| -------------------- | ------- |
| @material-ui/pickers |         v3.2.4|
| material-ui          |         v4.3.3|
| React                |         16.9.0|
| Browser              |         chrome|
| Moment         |         2.24.0|

## Expected behavior
The UI should work without it breaking.
![Screenshot 2019-08-29 at 14 46 51](https://user-images.githubusercontent.com/1423218/63942135-5f434180-ca6d-11e9-974e-b08871eff2db.png)
The both columns should have the grid value of `6`.

## Actual behavior
![Screenshot 2019-08-29 at 14 44 20](https://user-images.githubusercontent.com/1423218/63942077-46d32700-ca6d-11e9-81f2-7958cd19239a.png)
At the moment the breaking layout part has a grid value of `5`.

It seems that for some reason the `grid` elements do not occupy the whole space that they could.

Basically, if the month names are longer then planed (in the german language they receive a (.) dot or March is written without the abbreviation) then the layout gets broken like so.

Aside from that, how would I target those black icons on the blue background to change their color to white?

Thanks!

**Comment by lancee** [comment]
Not only German, I think this was caused by 

The `min-width` of `.MuiPickersModal-dialogRootWider` reduced to 310px (before it was 325px

**Comment by mnemanja** [comment]
Thanks!

As for my question, is it even possible to change the color of those icons? Before the update they were white.
